client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -116,7 +124,9 @@ class StatsPostDetail extends Component {
 	}
 
 	hasDontSendEmailPostToSubs( metadata ) {
-		return metadata?.some( ( { key } ) => key === '_jetpack_dont_email_post_to_subs' );
+		return metadata?.some(
+			( { key, value } ) => key === '_jetpack_dont_email_post_to_subs' && !! value
```

Comment:
@simison I think we should check the value here as well, because I've seen heaps of post which definitely have emails out already, but the meta exists with value of an empty string ''.

Reply:
@kangzj you can check with Loop team on changes on Newsletters/subscriptions UX if needed. Thanks! :-)